docs_src/security/Ch-RemoteDeviceServices.md

```diff
@@ -287,7 +287,6 @@ device services:
         - edgex-redis
         - edgex-security-spire-server
         - edgex-security-spiffe-token-provider
-        - edgex-vault
```

Comment:
see the edgex-compose
there should be a replacement, such as edgex-secret-store

Reply:
updated